### DISCUSSION TITLE
How to set environment variables when using local development?

### DISCUSSION THREAD

**Discussion by LydiaLane** [supabase/supabase#21530] [Questions] [answered]
I am trying to test manual linking on accounts:
https://supabase.com/docs/guides/auth/auth-identity-linking

It says I can set the environment variable GOTRUE_SECURITY_MANUAL_LINKING_ENABLED: true when self-hosting.

But HOW? How do I do that?
Thanks.

**Comment by LydiaLane** [comment, marked as answer]
Answer; needed to update to supabase beta release's cli to get the updated config.toml

**Reply by elsbree** [reply]
In case anyone ends up here looking for the relevant line to add to their config.toml like I did, you need to add the following in the `[auth]` section:
```yaml
enable_manual_linking = true
```
